org.eclipse.scout.rt.dataobject.test/src/main/java/org/eclipse/scout/rt/dataobject/id/AbstractIdCodecTest.java

```diff
@@ -762,7 +762,7 @@ public void testSignatureIdWithoutPassword() {
   }
 
   @IdTypeName("scout.FixtureDateId")
-  protected static final class FixtureDateId extends AbstractRootId<Date> {
```

Comment:
Move FixtureDateId and FixtureBooleanId to src/main/.../fixture?

Reply:
FixtureDateId moved, FixtureBooleanId added.